### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by andjos** [sonroyaalmerol/m3u-stream-merger-proxy#58]
## Describe the bug
Parsing error of m3u file in v0.7.0, no filters applied.

## To Reproduce

1. Normal first run

## Expected behavior
Clean parse of m3u

## Actual behavior
creates unwanted db entreys

## Container/Host Logs

```log
2024/03/20 19:43:31 Creating new database entry: h_256,w_256,f_auto,q_auto:eco,dpr_1/nqnvign9eowfvnqae4zf" group-title="Norway",Rikstoto Direkte 2 HD NO [Live During Events Only]
dpr_auto,f_auto,h_500,q_auto,w_500/v1/Assets/KLT/DNA%20TV/Kanavapaketit/ohjelmakirjastot/V%20kanavat/VFilm_family" group-title="Norway",V Film Family HD NO [Multi-Sub]
2024/03/20 19:43:31 Creating new database entry: dpr_auto,f_auto,h_500,q_auto,w_500/v1/Assets/KLT/DNA%20TV/Kanavapaketit/ohjelmakirjastot/V%20kanavat/VFilm_prempiere" group-title="Finland",V Film Premiere HD FI [Multi-Sub]
2024/03/20 19:43:31 Creating new database entry: /9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMSERUQEBAVEBAPERAQGBYVFxEVFhcXFRgWFhUSFRsYHSggGBolGxcYITEiJSkrLjEvFyAzODMtNygtMCsBCgoKDg0OGhAQGy0mICUvLS0tKy0tLS0rLS0tKy0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIAOAA4QMBIgACEQEDEQH/xAAcAAEAAgMBAQEAAAAAAAAAAAAAAQcCBQYEAwj/xABJEAABAwICBAgJCQcDBQEAAAABAAIDBBEGEgUhMVEHE0FhcYGRshciMjRTc6Gx0hQzQlJUYnKTwhYjJDWjwdFDgpJjosPw8RX/xAAaAQEAAwEBAQAAAAAAAAAAAAAAAQQFAwIG/8QANhEAAgECAwQIBAYCAwAAAAAAAAECAxEEEiEFMUFxE1FhgaGxwdEykeHwFBUiMzRSovEjNbL/2gAMAwEAAhEDEQA/ANMiIvtj5AFQUJUFAgVipusSoZ6ChSsbqCSFJRQoJCxWVlIiKjeL2Pmi+4iHSswLbFORkZ0eYRk8iyEG8r0IvWREZmfIQDpWYYNwWSKUkiLsIiIQEREAREQBERAEREAREQBCixJQAooKi6HohEQNKgEFRbcvoGLNLENnyEZ6F9BGFkoU2RFxZERSAiIoAREQBERAEREAREQBERAEREAREQBERAQVClQGoEQUDVkApSxNyA1SiIQEREAREQBERAEREAREQBERAERFNiAiIoJCIiAIiIAiIgCIiAWREQBERAEREARFKlEGJKZhvV2YboIXUsDzDGXOghJORlycouTq23US6U0e0lrpKYOaS0g8XcEGxB1atayPzX9TUabdu36M1Py3RN1Er9n1KUBUq7o6WhqgcrKecD6ojJHWNYXIYswI1jHT0l7MBc6Iku1cpYTr1bbHqXSltSnOWSacX27vvuPFXZ1SMc0Gpct/33nAIrPwJoSmlomSSQMkeXyAuIBJs429i6L9lqP7JF/xCirtWnTm4OL0duHAmns2c4KSktVfiUcivD9l6L7JF/xU/svR/ZIv+IXj84pf1fge/wAqqf2XiUrQ0j5pGxRDNJIcrRe2vbrJ2Bd1TYHpqdgl0hUAfdByM6LnxnHosvJi3LQ18LqSFrXNhDg0A2LnmVgNhrJ2alstG4KkqHfKNJSuc92viwdYG5xHk/hb2qcRiW4RnnyRkuFnJu+5dx5o4dRnKGXNJPlFLtPi7TOh4tTKXjLcvF39shBUDEGiJNUlHkvqvxTdXWw37F1Y0bQUw1xwR878mY9btZUspaCoFmsppvwiInqtrCz+no77VOef6WLqoVd14csv2yndK8Xx0nEfM8Y7Jt8m/i7dezevIvfp6BsdTNGwZWMlka0bgCQAvAvo4O8E+xeRhTVpNdr8wiIvR5CIiAIiIAiIgCIiAIiIAiIgCIikhl64X8zpvURd0KmNO+dT+vn77lc2F/M6f1EXdCrbSmDq188r2wXa+WV4OePY55IO3cVg7Pqwp16mZpc+Zt4+nKdKnlV/9HNUdU+F4licWSMNwR7jzbwr6o5c8THkWztY627MAbe1VzoDg+lc8Oqy1kbSDkBDnOt9EkagOgld7prSkdLE6WQgBosByuPIxq8bUrU60oxpavVXXbay9Sdn0p0YylU0Wm/xKZ0yHRVE0THkMZLKAAXAAZjbV0W7F6cKzvNbTgvJBlZtc5ampmL3ukd5Uj3PPS4kn3rZ4T8+pvXNW1ONqLT329NfEyoO9VW3Zl/6LTxy4ignIJBDWaxqPltVN/KX/Xd2u/yrkx3/AC+f8Le+1UqqGxknSfP0Lm1JWqrl6licG+h896yYl5uWRZiTa3lPF+c2HQ5bXHmJzStEMJHHytzX25GbM3STs6CtzhWAMoqdo9Cw9bhmJ7StZpjBENTM6eSaYOflFmmOwAFgBdhP/wBWb01OeKc63wrhbq3L1fWXuiqQw6hS+J8ee9lRzSue4ve4ve43LnEknpJWA231XVpeDOl9PP2w/AucxphSKiijfFJI8ySFhzllrZSdWVo3Lbp7QoVJKEG7vdpoZVXA1oRc5padpyLnEm5JJOsk6yecqERXSoERFACIiAIiIAiIgCIiAIiIAiIgCIilEMvTC/mdN6iLuhaTD+Kw+olpJyBI2aVkbtgeGucAw/eA7ffvML+Z0/qIu6FTOnD/ABU/J/ETH+o6xXzmFw0MROrGXc+rU38VXlRhTlHlz0Lm06agQl1JkMo12eCbjlDbEWd06lS+l9IzzyF1S9zntJbZwtlPK0N+irFwRi3jwKeod+/A8Vx/1AP1j29q+2NMJCpBnhAbUDqEg3H7249S9YSosHV6OtFLql98PI54qDxVJVKTv2ffH76ip1t8J+fU3rmrVSxua4te0tc02IOog7iFtcJefU3rWLdrP/jlyZk0f3I815otPHf8vn/C3vtVLK6cd/y+f8Le+1Uqs3Y/7L5+he2r+7Hl6lzYErhLQxWPjRDiXcxZs/7bHrXJcIFRVU9TmZUSshmaHNDXvDQQAHNHIOQ/7lqMF4j+RynPcwS2DwNZaRskA5uXeOgK06+igrYMrrSRPAc1zTsPI5p5CqdSP4TFZ5RvB37d/qmWacvxOHyRdpL09Ginf/36r7XN+Y//ACvPWaTmlAbNM+VrTmAe4usbWuL8y6jSPB1UsceIeyZnJc5HdY2dd186Tg7q3Hx3MiG8uLj2NGvtC1I4nB2zJx+WvuZ7w+JbytP56fM5BF7dM0HyeeSDNm4p2XNa19QN7cm1eJXIyUkmtz1Ksk4tphERSQEREAREQBERAEREAREQBERSAiIgO90fwiCKKOIUpPFRsjvxg15WgX8nmXF6SqRLNJKG5RLI99r3tmN7e1eZFXpYWnSk5QWr7WdqmIqVEozei3EtJBuDYg3BGog7wu40XwjSRxhk8PHubqzhwaSOTNqOvnXDKV6rYenWVpq5FKtOk7wdjpsS4jgrBmNIY5gLCQPF+hwy+MPatLoSvFPURzlufinF2W9r6iBr614kSFCEKfRx3dV2/l1ETqylPO953OmsfNqIJIDTZeNYW3zg23G2VcMilRRw9OinGmrImrXnVd5u75WIW00Np6opTeCQhpNyw+Mw9R2dIsVq1K6ygprLJXRzjJxd4ux39LwmPAtLStcd7Hlo7CD71lPwmm37ulF/vP1exqr5FT/LcNe+Xz9y0sdiLWzeC9j06UrnTzPneAHSuzENuBsA1X6F5VKhXYxUUktxVbbd2ERFJAREQBERQAiIgCIiAIi9OjaF88rYYhd8hsNw3uO4Aa0clFNvcEm3ZGNHSSTPEcTDI92wN29PMOcruNE8G7nAOqZsn3Ixc9bjq7AeldjhzQMVHHkYLvIGd5AzOP8AYbgtZiDG8FM4xsBnmbqIabNadznb+YXWFU2hXrzyYde/PsNingqVGOeu+7h9SGcH1GBYte7nLyPdZeOt4NoCP3M0kbvvZXt7NR9q0UvCPVE+LHE0brPPtzLYaM4StYFTDYfWjJNulp/yo6HaMP1Zm+zNfwZKq4GWll8rHMaewtUUmuRofF6Rly3mzcrfdzlaNX5SVMVRHnY5ssUg3XB3gg9hBVZY6wt8ldx8I/h3mxHo3Hk/CeTd2Kzgto9JLo6qtL74cGV8XgejXSU9V98eKOQXVYIwzHW8aZXuaIiwAMyi+a5uSQdy5VWTwSeTUdMP61ax9SVOhKUHZ6a96OGCpxnWUZK618jW4rwNxEXHUznyNZre11iQ367bAXtyhcQv0QW3261UePML/Jn8dC3+HlOwf6bj9H8J5Ozcqezse5voqr14Pr7C1jsGoLpKa04r1OTREWwZZ0WDMPMrZXske5jY2B3i2uSTblWwxlg+OjgbNHI595BGQ/LsIcbiwG5ergl+en9UzvFbvhT8yHr2d2RY1WvUWOVNS/TdadyNSlQpvBubWtnryKoXdYYwGJ4WzzyFglGZrWZb5TsLib7dtudcKr2w15nT+oi7oXbaeIqUqccjtd+hy2fQhVm86vZHN+DSn9NL/T+FZDg1pfSy9rPhXjxNjienqpII4oy2PKLuzkm7Q7kI3rWeEmp9DD2SfEqkIbQnFSUtHrvRalUwMZOLju7GdB4Nqb0kvbH8KeDam9JL2s+Fc/4San0MPZJ8SeEmp9DD2SfEvXQbR/t/kiOmwH9f8WaTFuio6WpMETnOa1jHHNa4Lrm2obrHrWlXr0ppB9RK+aS2eQ3NtQFhYAdQC8i16UZRhFTd3bXmZVRxc24qy4BERdDwEREAREQBWXwW6JAjfVuHjSExs5mtPjEdLu6q0V44ThDKKnaPQscelwzH2lZm1qmWjl62aOzaalVcupfQ1HCDp408IiidaaouLja1g8pw59YA6TuVSrq+EqoLq1zTsiZGwdYznvLlF12dRVOhF8Xq/vyOOOqudZrgtEFKhFfKh0OCtPupZwCf3Ezg145ATqEg3EcvNfmVu6Ro2TxuheLskaWnr2Ec42qgCFfGHagyUsEh2vhjJ6coWDtekoyjVW97+7VM2Nl1HKMqb3L13lHVtK6KR8T/AConuYf9ptdWHwS+TUdMP61zXCFCG6Qlt9NsT+stAPuXS8Evk1HTD+tWsZPpMFn61F+KK+EhkxeXqv5M6bTunmUr4RLqjne5hffUywFiebXrPItlVU7JWOjeA+ORpaQdYIK4jhb+bg/FL3Qvjwd4o2Uc51jVE48v/SP9uzcsr8G5YZVob1e/dx7uJpfiksQ6UuNrd/A5XFOgHUc2Q3dG+7o37x9U/eHL2rSq9dP6IjqoTDJy62u5WuGxw/8AdhKpXSujpKeV0Mos9h6iORzeYrZwGM/ERtL4l49vv9TKxuE6GV4/C/DsOw4Jfnp/VR94re8KfmQ9ezuvWi4Jfnp/VR94re8KfmQ9ezuvVGt/2MecfJFyj/AfKXqVOVe+HPNKf1EXdCogq98OeaU/qIu6F22x+3DmctlfHLkvMqjHv8wn6Y+41c+rp0jg+knldNKxxkfa5D3gagBsBtsAXm/YCh9E/wDMk/yoo7Uowpxi09EluXBW6xV2dWlNyVtW3vfsVAoVv/sBQ+if+ZL/AJWsxNg+ihpZZmRuD2MOU8Y8+MbAaibHWV2jtWjKSilLXTcvc5S2bWim3bTtfsVmiItIzwiIhIREQBERAFeGEpw+ip3DkiYw9LBlPtCo9WPwW6XBa+kcfGaTKznafLA6DY/7lmbVpOVDMuDuaGzaijVs+KNLwm0pbW5+SaNjh0t8Uj2DtXJK48caB+VwDiwONhJez73I6Pr94Cp17SCWuBDmkggixBG0EchXTZtdVKCjxjo/RnjH0nCq3weq9SERFfKQV9aCpjFTQxHbHFG09IAuqwwFh11RM2eRv8PC4O1/TeNYaN4BsT0WVpaUrWwRPmebNjbm6TyAc5Nh1rB2tWU5xpR1a383uXP3NnZlLLGVWW57uS3sqXhAnD6+W30BGzrDRf3rpuCXyajph/8AIq7q6h0kjpX+VI5zz0uN1YnBL5NR0w/rVzGw6PBZOpRXiithJ58Xm67vwZPC183T/il7oVbA2NwbEa7jaOcKyeFr5uD8UvdCrVdNm/xo9/mc9ofyHyRbmBsTfKo+KkP8REBfkzt9IOffz9K9GMcOCriu2wqI7ljtl98ZO4+wqo6GsfDI2WJ2V8ZuD7wd4I1W51dGG9OMrIRI3U4eK9nK127oO0FZmNw0sLUValovJ9XJ+6L+ErrEQdKpv817pnGcFcZbUVDXAtc2NjSDtBDjcFbvhT8yHr2d166GHRcbKh9S0WklY1jrbDl2OPPydQXPcKfmQ9ezuvXiFdVsbCpbe189DpKi6OElDqUvUqcq98N+aU/qIu6FRBV74c80p/URd0K7tj9uHMqbL+OXJeZweLMV1cFXLFFKGxsLbDJGdrWnaRvK1P7d13pv6cPwr549/mE/TH3Grnlaw+FoujBuCvZcF1IrV8RVVSSUnvfmdL+3dd6b+nD8K8elMUVVRGYppc0ZIJAaxt7axcgb1pkXeOGoxaagr8kcpYirJWcnbmSoRF2OIREQBERAEREAX3oat8MjZYnZXsIcD/Y7xyda+CI0mrMJtO63l14YxHHWR3BDZWjx4ydY5xvbzrDT2E6er8d4Mcv12WBP4gdTvfzqm4J3McHscWPabhzSQQeYhdlorhFnYA2eNs4GrMDkf0nVY+xYdXZtWlPPhn3Xs137mvtmvSx9OpHJXXqn7Hol4MpL+LVMI52OB9jitjorg5iYQ6oldPb6LRkb16yT2hZt4Sqa2uGcHmER/WvJWcJjbfuaZxO+RzQB1NvftC8N7Sn+nXwXie0sBH9V14vwO5vHDH9GKKNvM1rWj3BVTjfFPyt/FREinjNxtBkd9Y824dfRqdN4gnqz++f4oNwxvisHPblPOVq1bwWzuhfSVHeXDs7+LK2Lx3SLJBWXn9CVZHBL5NR0w/rVbLoMLYodQ8YGxCUS5SQXFpBbe1jY71ax1KVWg4Q36eZXwdSNKspS3a+TOq4Wvm4PxS90Ktl0GKcUurQwOiEQiLjqcXEk2G2w3LnlGBpSpUFGa11GMqRqVnKO7QLaYc00+jmErLlp8V7ORzd3SOTnWrRWpxjOLjJXTOEJOMlJb0X/AEFayaNssTszJBcH+x5+Rcxwp+ZD17O69cNhfFUlFma1glifryFxbZ31mmxtcbegL0Ynxi+tibCYGxNa8SXDy8kgEAeSLDxvcsOls6rSxMZLWKe+63eZr1MdTqUGn8TW63HyOYV7Yb8zp/URd0KiV1eH8cTUsQhMbZmM8m7i0tG3LexuFd2jhp14RUN6fLgVMBXhSm3Pc0dXp7Ajamd9Rx7ozJYkZQ4XADbjWOQBeDwYt+1u/LHxL4eE5/2Nv5p+BR4Tn/Y2/mn4FShDaMYqK3LtiW5TwMm21q+yR6PBi37W78sfEp8GLftbvyx8S8/hOf8AY2/mn4E8J7/sbfzT8C9W2l1+MSL4Dq8JHK4n0L8jn4jPxgyNkDrW1OuLEXPKCtStniHTDqyczOYGeI1gaCTYNudp27StYtejn6OPSfFbXn3aGVVyZ3k3cOQREXQ8BERAEREAREQBERAEREAREQBSoRSAiIgCIiAKVCKAFKhFIJRQiEWJRQiCxKhEQkIiKAES6JcBERAEREAREQBERAEREAREQBERAEREAREugCLF0gXzM24JmRNmfZCV5y871ivOY9ZD7mQLEzcy+SLzmYyn0MpUcYd6xUJdk2RlmO9AsVkFIChSigHoREXQ5hERAEREAREQBERAEREARF83ygc6XB9Fi54C+DpSeZYBeXPqPagfZ024L5ucTtUKCvDdyUkjJQiKCQiIpAREQGShSoUoBZBYrIIGEREIP//Z" group-title="United Kingdom",RTÉ One HD UK

```

## Additional context
Tested in 0.6.1 works ok.

**Comment by sonroyaalmerol** [comment]
Hmmmm. I can see where the parser went wrong, but what's weird to me is the fact that you said it works on 0.6.1. Nothing should've changed with the parser logic between 0.6.1 and 0.7.0.

Well, either way, can you do me a favor and check what attribute is before `group-title` in your source M3Us? I'll need to properly parse that to fix this.

**Comment by sonroyaalmerol** [comment]
Can you test the `:dev` build to see if I was able to fix it?

**Comment by andjos** [comment]
`:dev` looks fine, checked the db and every think looks normal.

**Comment by sonroyaalmerol** [comment]
I've released the fix in 0.7.1. Thanks for testing it out!
